samples/llama_index_quick_start.ipynb

```diff
@@ -0,0 +1,1014 @@
+    "\n",
+    "# Please fill in these values.\n",
+    "region = \"us-central1\"  # @param {type:\"string\"}\n",
+    "cluster_name = \"my-cluster\"  # @param {type:\"string\"}\n",
+    "instance_name = \"my-primary\"  # @param {type:\"string\"}\n",
+    "database_name = \"my-database\"  # @param {type:\"string\"}\n",
+    "password = input(\"Please provide a password to be used for 'postgres' database user: \")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "T616pEOUygYQ"
+   },
+   "source": [
+    "### Create an AlloyDB Instance\n",
+    "If you have already created an AlloyDB Cluster and Instance, you can skip these steps directly to the \"Connect to our New Database\" section."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "xyZYX4Jo1vfh"
+   },
+   "source": [
+    "> ⏳ - Creating an AlloyDB cluster may take a few minutes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "XXI1uUu3y8gc"
+   },
+   "outputs": [],
+   "source": [
+    "# Quick input validations.\n",
+    "assert region, \"⚠️ Please provide a Google Cloud region\"\n",
+    "assert instance_name, \"⚠️ Please provide the name of your instance\"\n",
+    "assert database_name, \"⚠️ Please provide the name of your database_name\"\n",
+    "\n",
+    "# create the AlloyDB Cluster\n",
+    "!gcloud beta alloydb clusters create {cluster_name} --password={password} --region={region}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "o8LkscYH5Vfp"
+   },
+   "source": [
+    "Create an instance attached to our cluster with the following command.\n",
+    "> ⏳ - Creating an AlloyDB instance may take a few minutes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "TkqQSWoY5Kab"
+   },
+   "outputs": [],
+   "source": [
+    "!gcloud beta alloydb instances create {instance_name} --instance-type=PRIMARY --cpu-count=2 --region={region} --cluster={cluster_name}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "BXsQ1UJv4ZVJ"
+   },
+   "source": [
+    "To connect to your AlloyDB instance from this notebook, you will need to enable public IP on your instance. Alternatively, you can follow [these instructions](https://cloud.google.com/alloydb/docs/connect-external) to connect to an AlloyDB for PostgreSQL instance with Private IP from outside your VPC."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "OPVWsQB04Yyl"
+   },
+   "outputs": [],
+   "source": [
+    "!gcloud beta alloydb instances update {instance_name} --region={region} --cluster={cluster_name} --assign-inbound-public-ip=ASSIGN_IPV4"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "mjA6AiAzB2Du"
+   },
+   "source": [
+    "Now create a connection pool to connect to our instance."
```

Comment:
Move this section and code snippet below header "Create a Database" to group code snippets